src/corner_kick/src/store/actions/ros.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * This file specifies ROS specific action
+ */
+
+import { createAction } from 'typesafe-actions';
+
+/**
+ * Connect to ROS
+ */
+export const start = createAction('ros/START');
```

Comment:
The `ros/` scoping is implicit, not explicit, so I _think_ `ros_START` would be more clear

Reply:
Technically specific to Redux, but I implemented the change